public/components/GeoPlot.vue

```diff
@@ -752,9 +755,11 @@ export default Vue.extend({
         }).$mount();
 
         // Add interactivity to the layer.
-        layer
-          .bindTooltip(tooltip.$el as HTMLElement)
-          .on("click", () => this.showImageDrilldown(imageUrl, item));
+        layer.bindTooltip(tooltip.$el as HTMLElement).on("click", () => {
+          if (this.isRemoteSensing) {
```

Comment:
You should put the `isRemoteSensing` test outside the `on("click...`

Reply:
Will do.